cmd/store/store.go

```diff
@@ -321,19 +413,116 @@ func (s *Store) ListEntries(name ...string) ([]string, error) {
 	return entries, nil
 }
 
-func (s *Store) StoreClaim(data []byte) error {
+func (s *Store) ClaimType(data []byte) (*jwt.ClaimType, error) {
 	// Decode the jwt to figure out where it goes
 	gc, err := jwt.DecodeGeneric(string(data))
 	if err != nil {
-		return fmt.Errorf("invalid jwt: %v", err)
+		return nil, fmt.Errorf("invalid jwt: %v", err)
 	}
 	if gc.Name == "" {
-		return errors.New("jwt claim doesn't have a name")
+		return nil, errors.New("jwt claim doesn't have a name")
+	}
+	return &gc.Type, nil
+}
+
+func (s *Store) syncRemoteAccount(u string) (int, []byte, error) {
+	c := &http.Client{Timeout: time.Second * 5}
+	r, err := c.Get(u)
+	if err != nil {
+		return 0, nil, fmt.Errorf("error loading %q: %v", u, err)
+	}
+	defer r.Body.Close()
+	var buf bytes.Buffer
+	_, err = io.Copy(&buf, r.Body)
+	if err != nil {
+		return 0, nil, fmt.Errorf("error reading response from %q: %v", u, err)
+	}
+	return r.StatusCode, buf.Bytes(), nil
+}
+
+func (s *Store) pushRemoteAccount(u string, data []byte) (int, []byte, error) {
```

Comment:
same comment on URLs

Reply:
the coordination method extracts these, the others only do the needy http